packages/components/src/components/hds/layout/grid/index.ts

```diff
@@ -24,11 +24,20 @@ export const ALIGNS: HdsLayoutGridAligns[] = Object.values(
 export const DEFAULT_GAP = HdsLayoutGridGapValues.Zero;
 export const GAPS: HdsLayoutGridGaps[] = Object.values(HdsLayoutGridGapValues);
 
+type ResponsiveColumnWidths = {
+  sm?: string;
+  md?: string;
+  lg?: string;
+};
+
 export interface HdsLayoutGridSignature {
   Args: {
     tag?: AvailableTagNames;
     columnMinWidth?: string;
-    columnWidth?: string;
+    columnWidth?:
+      | string
+      | ResponsiveColumnWidths
+      | [string, ResponsiveColumnWidths];
```

Comment:
Do we need to have a default value here? I feel like if youre going to the effort of adding responsive column widths, you don't need the default too.

Another reason I bring this up is that I think it makes the API kind of confusing. I'd rather it just take an object with breakpoints + values

Reply:
I simplified the API to remove the array